src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs

```diff
@@ -1,345 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities
-{
-#if NET451
-    using System;
-    using System.Collections.Generic;
-    using System.Diagnostics;
-    using System.IO;
-    using System.Reflection;
-
-    using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
-
-    /// <summary>
-    /// Implementation of finding assembly referencies using "managed route", i.e. Assembly.Load.
-    /// </summary>
-    public static class AssemblyHelper
-    {
-        private static Version defaultVersion = new Version();
-        private static Version version45 = new Version("4.5");
-
-        /// <summary>
-        /// Checks whether the source assembly directly references given assembly or not.
-        /// Only assembly name and publickey token are match. Version is ignored for matching.
-        /// Returns null if not able to check if source references assembly.
-        /// </summary>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
-        public static bool? DoesReferencesAssembly(string source, AssemblyName referenceAssembly)
```

Comment:
This is public API, Can be used by old adapters. [Example](https://github.com/Microsoft/testfx/blob/abd5ea3585657914490c83d5d2b4c9e732640a89/src/Adapter/PlatformServices.Desktop/DesktopTestSource.cs#L46)